File: HW5/程式作業/AddThePage.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>
#include <math.h>
#include <string.h>


int main(){

    long long int n;
    scanf("%lld",&n);
    while(n--){
        long long int sum;
        scanf("%lld",&sum);
        long long int i=0;
        while(sum >= i+1){
            i++;
            sum -= i;

        }
        i+=1;
        printf("%lld %lld",i,i-sum);
	if(n >0 ){printf("\n");}


    }



    return 0;
}

File: HW5/程式作業/AddThePage.py

n = input()
for i in range(0,int(n)):
     sum = input()
     sum = int(sum)
     j = 0
     while sum >= j+1:
          j += 1
          sum -= j

     j += 1
     print(j-sum," ",j, end='')
     if i != int(n)-1:
          print("")

